### PR TITLE
Add ARM64 Build of Docker Image

### PR DESCRIPTION
**Pull request by mabentley85**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | Adds ARM64 build of the docker image

### CODE REVIEW

**Comment by schmunk42** [comment]
The build itself is nice, but I think we need a way to trigger it just on a weekly basis and/or manually, just because the build runs over an hour - I think we don't want that for every commit :)

**Comment by mabentley85** [comment]
@schmunk42 Agreed, these jobs take way too long.

Running them only when manually triggered or once a week could be accomplished by updating the `on` section to look like this:
```
on:
  schedule:    
    - cron: '0 6 * * 1'
```
Building the master branch on Mondays at 6AM (for example). Though the pushing to docker hub would only occur when the current revision has been tagged.

What do you think of that?

**Comment by schmunk42** [comment]
@mabentley85 Sounds great.

We already have a weekly schedule for this images.

Maybe some jobs can only run `if: schedule,manual,[extended build branch]` (pseudo-code) or we create another github-workflow with only `schedule`. I haven't tried this yet, so feel free with the implementation.

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ name: Docker Image CI
 
 on:
   workflow_dispatch:
-  push:
```
Comment (schmunk42) [inline .github/workflows/docker-image.yml:5]: You removed `push` and `PR`, but we need them.
Maybe it's better to create two separate workflows (.yml) files.

One like the existing for `push` and `pull_request` the other for `dispatch` and `schedule` with ARM builds.

**Review by schmunk42 — CHANGES_REQUESTED** [review]

**Comment by mabentley85** [comment]
@schmunk42 Closing this Pull Request as the solution provided by @albertborsos in #33 is better.

**Pull request closed without merging**